providers/src/airflow/providers/openlineage/plugins/listener.py

```diff
@@ -325,15 +338,22 @@ def on_failure():
                 logical_date=dagrun.logical_date,
             )
 
+            if not task_instance.queued_dttm:
+                # This should not happen outside of tests, but if it does, we should log it and continue
```

Comment:
why should we log it?  does it really matter?  i would not bother if it's truly unexpected. wouldn't it cause problems and be surfaced elsewhere?  in that case, this is just noise.

Reply:
Removed.